### ISSUE TITLE
Default labels not in Favourite or Recent list

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1390]
I just did a fresh install and I have the new default labels which is good. However when I added a bookmark those labels were not visible in the first label assign dialog. I had to go into the label list to find them. Is that by design or did something go weird with my install?

What I think should happen is that the default labels should all be assigned as either Recent labels or Favourite labels so they are accessible from the quick label assigning form. Perhaps...
- Blue and Green could be assigned to favourites
- Underline as Recently used
- Red left untouched so they can find it in list

I would also support adding a few more colours - Yellow an Orange would b good.

![image](https://user-images.githubusercontent.com/13920678/126857063-575fd4fd-0119-444b-86d3-23076a7ba502.png)

**Comment by tuomas2** [comment]
Hmm, I implemented this to have them all as favourite, but apparently there's a bug then.

**Comment by agrogers** [comment]
Don't chase it yet. Let me try again. AB kept crashing the first time i
tried to download documents - maybe it caused some issue there.
I will post again if i can reproduce it.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)
